OH-Archive/assets/js/indexScript.js

```diff
@@ -122,7 +122,7 @@ request.onload = function () {
                 const h4 = document.createElement("h4");
                 h4.setAttribute("style","text-align: left;");
 
-                const h4Text = document.createTextNode(general.isSubjof);
+                const h4Text = document.createTextNode(general.isSubjof.replace(/(")([^"]*)(")/g, '«$2»'));
```

Comment:
I don't like this.
The mistake is in the data. If we change the string here it will lead to confusion. The Data itself has to be changed... I did this in omeka.

Revert this change.